frontend/encore/cdn.rst

```diff
@@ -1,9 +1,7 @@
 Using a CDN
 ===========
 
-Are you deploying to a CDN? That's awesome :) - and configuring Encore for that is
-easy. Once you've made sure that your built files are uploaded to the CDN, configure
-it in Encore:
+When deploying to a CDN, your uploaded built files need to be configured in Encore:
```

Comment:
this takes away a lot of ryan's enthusiasm, which is a pitty. can we keep more of this intact and only get rid of the "easy" bit?

Reply:
done